docs/getting_started/gateway.md

```diff
@@ -0,0 +1,15 @@
+---
+title: Gateway
+sidebar_position: 9
+---
+
+The gateway is a reverse proxy service that is used to route incoming requests from clients to the intended server. It acts as an intermediary layer between the client and the Private Shard infrastructure including the RPC and GraphQL Indexer endpoints. It provides additional security, privacy, and scalability. The gateway also acts as a permissions management ("firewall") service allowing users to filter between the public clients and the Private Shard. It is monitoring the traffic that comes into and out of the computer, and blocking any malicious or unauthorized activity.
```

Comment:
`service allowing users to filter between the public clients and the Private Shard`
What does this mean? Who are the users, what are they filtering?